src/EnergyPlus/Vectorize.hh

```diff
@@ -0,0 +1,34 @@
+#ifndef EnergyPlus_vectorize_hh_INCLUDED
+#define EnergyPlus_vectorize_hh_INCLUDED
+
+// EnergyPlus Vectorization Support
+
+// ObjexxFCL Headers
+#include <ObjexxFCL/vectorize.hh> // ASSUME_ALIGNED and RESTRICT macros
+
+// C++ Headers
+#include <cassert>
+#include <cstdint>
+
+// EnergyPlus headers
+#include "EnergyPlus.hh"
+
+namespace EnergyPlus {
+
+	int const VEC_LENGTH = VEC_ALIGN / sizeof(Real64);
+
+#define ROUND_TO_VEC_LENGTH(I) (((I) + ((VEC_LENGTH) - 1)) & ~((VEC_LENGTH) - 1))
+
+
+#define EXPLICIT_VECTORIZATION
+
+	// EXPLICIT_VECTORIZATION assumes that vectors are aligned,
+	// which is not a valid assumption on 32-bit builds
+#if defined(_WIN32) && !defined(_WIN64)
```

Comment:
@amirroth Why is it not a valid assumption for 32-bit builds, but it is for 64-bit builds? Are we just getting lucky in 64-bit builds with the (much) larger memory address space compared to 32-bit?

I know this is a (slight) performance hit, but why not use [_mm_loadu_pd](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#techs=SSE,SSE2&text=_mm_loadu_pd&expand=3106,3106)? This allows you to load unaligned data from memory. From my research, it seems like modern processors have a much smaller performance hit using this function versus `_mm_load_pd` than they used to.